src/toil/cwl/cwltoil.py

```diff
@@ -3218,38 +3306,13 @@ def main(args: Union[List[str]] = None, stdout: TextIO = sys.stdout) -> int:
             fill_in_defaults(tool.tool["inputs"], initialized_job_order, fs_access)
 
             for inp in tool.tool["inputs"]:
-
-                def set_secondary(fileobj):
-                    if isinstance(fileobj, Mapping) and fileobj.get("class") == "File":
-                        if "secondaryFiles" not in fileobj:
-                            # inits all secondary files with 'file://' schema
-                            # later changed to 'toilfile:' if imported into the jobstore
-                            fileobj["secondaryFiles"] = [
-                                {
-                                    "location": cwltool.builder.substitute(
-                                        fileobj["location"], sf["pattern"]
-                                    ),
-                                    "class": "File",
-                                }
-                                for sf in inp["secondaryFiles"]
-                            ]
-
-                    if isinstance(fileobj, MutableSequence):
-                        for entry in fileobj:
-                            set_secondary(entry)
-
-                if shortname(inp["id"]) in initialized_job_order and inp.get(
-                    "secondaryFiles"
-                ):
-                    set_secondary(initialized_job_order[shortname(inp["id"])])
-
                 if (
                     shortname(inp["id"]) in initialized_job_order
                     and inp["type"] == "File"
                 ):
                     initialized_job_order[shortname(inp["id"])]["streamable"] = inp.get(
                         "streamable", False
-                    )
+                    )  # TODO also for nested types that contain streamable Files
```

Comment:
We should probably make an issue for this?

Reply:
Done :-) https://github.com/DataBiosphere/toil/issues/3801